shared/message/input/MotionCapture.proto

```diff
@@ -101,7 +101,7 @@ message MotionCapture {
     /// https://en.wikipedia.org/wiki/SMPTE_timecode
     uint32 timecode     = 3;
     uint32 timecode_sub = 4;
-    double timestamp    = 5;
+    double time         = 5;
```

Comment:
`time` probably isnt the best name either to he honest. Maybe `time_step`? Or `message_time`? @TrentHouliston any thoughts?

Reply:
What does it actually represent?

Reply:
I think it is the time since the software (motive) was started. So it is a timestamp

Reply:
What about `external_timestamp`? Or if this stuff is very specific to motive, then `motive_timestamp`? Or `received_timestamp`?

Reply:
I think the idea would be to try and sync up this timestamp with the system time. So maybe `motive_timestamp` or `natnet_timestamp` since this is the NatNet module.